### DISCUSSION TITLE
IndexError: list index out of range while processing TerraSAR with unpackFrame_TSX.py

### DISCUSSION THREAD

**Discussion by wandi12345** [isce-framework/isce2#409] [Q&A] [answered]
Hi everyone,
when I run this command unpackFrame_TSX.py -i /misc/TerraSAR/DATA/ -o /misc/TerraSAR/UPDATA/
this error occured: IndexError: list index out of range

The data structure looks like this：
/dims_op_oc_dfd2_679780118_1/TSX-1.SAR.L1B/TDX1_SAR__SSC______SL_S_SRA_20211012T103712_20211012T103714/
and, I also tried change unpackFrame_TSX.py lines 32, but not work.
![image](https://user-images.githubusercontent.com/74475233/144161655-81434db0-7d76-452d-95ed-d77727a0fa25.png)

**Comment by piyushrpt** [comment]
This is a python error. Try printing out results of glob, fname etc and debug. You are probably missing a `/` after TDX1`. I don't think this is an error in ISCE.

**Reply by wandi12345** [reply]
Thank you very much

**Comment by fdelgadodelapuente** [comment, marked as answer]
Hi, 

You can change the line to

    fname = glob.glob(os.path.join(hdf5,'T*/*/T*.xml'))[0]

and pass the dims directory.

Francisco

**Reply by wandi12345** [reply]
Thank you very much, You are right.
